### PR TITLE
Small fixes, decluttering + better build scripts

### PR DESCRIPTION
**Pull request by OmniTroid**
* Fixed a divide by zero crash if blip_rate is set to zero
* Improved the post build script for macos
* Decluttered the .pro file and bumped version to 2.6.1

all in all, no major stuff. shouldn't break anything

### CODE REVIEW

**Comment by OmniTroid** [comment]
+
* added icns file for macos and the icons used to generate it
* fixed bassopus plugin load

File: Attorney_Online.pro [hunk 1]
```diff
@@ -1,16 +1,9 @@
-#-------------------------------------------------
-#
-# Project created by QtCreator 2016-12-29T01:14:46
-#
-#-------------------------------------------------
```
Comment (oldmud0) [inline Attorney_Online.pro]: :(

Comment (OmniTroid) [reply Attorney_Online.pro]: do we really need to know when this project was created by qtcreator though

Comment (oldmud0) [reply Attorney_Online.pro]: yes

**Pull request merged**